content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,327 @@
+
+   - Add the :guilabel:`Fiscal Position` for :ref:`Avatax Brazil <brazil/fiscal-positions>`.
+   - :guilabel:`Tax Regime`
+   - :guilabel:`ICMS Taxpayer Type`
+   - :guilabel:`Main Activity Sector`
+
+   .. image:: brazil/contact-fiscal-configuration.png
+     :alt: Company fiscal configuration.
+
+#. Finally, upload a company logo and save the contact.
+
+.. note::
+   If you are a simplified regime you need to configure the ICMS rate in
+   :menuselection:`Accounting --> Configuration --> Settings --> Taxes --> Avatax Brazil`.
+
+.. _brazil/avatax-account:
+
+Configure AvaTax integration
+----------------------------
+
+Avalara AvaTax is a tax calculation provider that can be integrated in Odoo to automatically compute
+taxes by taking into account the company, contact (customer), product, and transaction information
+to retrieve the correct tax to be used.
+
+Odoo is a certified partner of Avalara Brazil, which means that workflows covered within the scope
+of the integration were reviewed by Avalara experts.
+
+Using this integration requires In-app-purchase (IAP) to compute taxes. Every time you compute
+taxes, an API call is made, using credits from your |IAP| credits balance.
+
+IAP (Odoo In-app purchase)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|IAP| gives access to additional services through Odoo. In the case of Brazil, it allows you to
+compute taxes to be used on sales operations.
+
+.. seealso::
+   If you want to know more about this service, you can visit the Odoo official :doc:`IAP
+   documentation </applications/general/in_app_purchase>`.
+
+The credits are consumed per each API call (tax computation) that is sent to AvaTax.
+
+Credential configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To activate AvaTax in Odoo, you need to create an account. To do so, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Taxes`, and, in the :guilabel:`AvaTax
+Brazil` section, add the email address you want to use to log in to the AvaTax portal, and the click
+on :guilabel:`Create account`. This email is used as the administrator email address in AvaTax.
+
+After you create the account from Odoo, you need to go to the Avalara Portal to set up your
+password.
+
+.. warning::
+   Only create an account with the AvaTax portal email you want to use in production when you are in
+   your production database (real transactions). If you want to test, you can use a fake email to
+   create the account with credentials for you to test on a testing or sandbox database.
+
+.. tip::
+   If you do not know how to log in to the AvaTax portal for the first time, `click here
+   <https://drive.google.com/file/d/1jPtKzilfLrfdRXyPGacQNr6i2zvQvoOd/view?usp=sharing>`_. You can
+   start using AvaTax in Odoo without creating a password and accessing the Avalara Portal; for
+   Odoo, the only requirement to start using the Avalara Tax Computation Engine is to create an
+   account from the settings page.
```

Comment:
It doesn't matter if something has to be configured outside of Odoo; we can - and sometimes should - still document processes outside of Odoo. Unless the external 'provider' documents it better (assuming here it would be better maintained) and/or it isn't necessary to configure the feature (i.e., extra info).
If the information makes too much noise, we don't need a link to an external document.

Here the information comes from us, meaning we could put it in the documentation instead. Furthermore, the information is relatively short at its core (the screenshots aren't needed):
> 1. Login on the [Avalara portal](https://portal.avalarabrasil.com.br/Login)
> 2. Click on “Meu primeiro acesso”
> 3. Add the email address you used in Odoo to create the Avalara/Avatax Account, and then click "Solicitar Senha"
> 4. You will receive an email with a Token and a link to create your password. Click on this link and copy-paste the token and allocate your desired password.

If you feel this information is superfluous, feel free not to include it, but I would avoid linking to an inconvenient and no so good-looking PDF.